### PR TITLE
mark other nodes in routing table on test-connect

### PR DESCRIPTION
**Pull request by whyrusleeping**
License: MIT
Signed-off-by: Jeromy <jeromyj@gmail.com>

### CODE REVIEW

File: routing/dht/dht_test.go [hunk 1]
```diff
@@ -78,6 +78,9 @@ func connect(t *testing.T, ctx context.Context, a, b *IpfsDHT) {
 	if err := a.host.Connect(ctx, pi); err != nil {
 		t.Fatal(err)
 	}
+
+	a.Update(ctx, b.self)
+	b.Update(ctx, a.self)
```
Comment (jbenet) [inline routing/dht/dht_test.go]: :-1:

**Comment by whyrusleeping** [comment]
@jbenet this is RFM

**Comment by jbenet** [comment]
@whyrusleeping might be nice to add a note why we need to loop and wait for future us. otherwise LGTM.

**Comment by whyrusleeping** [comment]
@jbenet commented!

**Comment by jbenet** [comment]
LGTM! :+1:

**Pull request merged**